incubator/hnc/pkg/reconcilers/hierarchical_namespace_test.go

```diff
@@ -57,24 +56,14 @@ var _ = Describe("HierarchicalNamespace", func() {
 	})
 
 	It("should set the hns.status.state to Conflict if the namespace's owner annotation is wrong", func() {
```

Comment:
Maybe specify why the owner annotation is wrong, e.g., IIUC, 
"should set the hns.status.state to Conflict if a namespace of the same name already exists"

Reply:
Yes, for the hns instance perspective, the conflict is only when the annotation doesn't match. It could be the reason you mentioned or someone with higher privileges changes the owner annotation. Here I wanted to introduce a mismatch to trigger the conflict.